app/consulting/index.tsx

```diff
@@ -1,40 +1,32 @@
-import { useRouter, type NextRouter } from "next/router";
+"use client";
+
+import { useRouter } from "next/navigation";
+
 import { useEffect, useMemo, useRef, useState } from "react";
 import { MdLiveHelp } from "react-icons/md";
 
 import { wrapGrid } from "animate-css-grid";
 
-import { client } from "@/tina/client";
-import { tinaField, useTina } from "tinacms/dist/react";
+import { tinaField } from "tinacms/dist/react";
 
-import { TODAY } from "hooks/useFetchEvents";
-import { InferGetStaticPropsType } from "next";
+import { Category } from "@/components/consulting/index/category";
+import { Tag } from "@/components/consulting/index/tag";
+import { Container } from "@/components/util/container";
+import { Breadcrumbs } from "app/components/breadcrumb";
+import { useSearchParams } from "next/navigation";
 import { ParsedUrlQuery } from "querystring";
-import { Breadcrumbs } from "../../components/blocks/breadcrumbs";
-import { Category } from "../../components/consulting/index/category";
-import { Tag } from "../../components/consulting/index/tag";
-import { Layout } from "../../components/layout";
-import { Container } from "../../components/util/container";
-import { SEO } from "../../components/util/seo";
 
 const allServices = "All SSW Services";
 
-export default function ConsultingIndex(
-  props: InferGetStaticPropsType<typeof getStaticProps>
-) {
+export default function ConsultingIndex({ tinaProps }) {
   const gridRef = useRef(null);
-  const { data } = useTina({
-    query: props.query,
-    variables: props.variables,
-    data: props.data,
-  });
-
+  const params = useSearchParams();
   const router = useRouter();
   const [selectedTag, setSelectedTag] = useState(
-    getSelectedTagFromQuery(router.query)
+    getSelectedTagFromQuery(params.get("tag") as unknown as ParsedUrlQuery)
```

Comment:
This is a bit weird... Can you try fix it, or at least add a comment explaining why you need it :)

Reply:
Fixed the issue.